Do not start ASCII file with "PACK", this confuses the 'file' command and the
'subst' robotpkg target as a consequence (the file is interpreted as a Quake
world extension ;)

--- build-aux/pkg-config.pc.in~	2010-12-07 16:41:53.000000000 +0100
+++ build-aux/pkg-config.pc.in	2011-10-10 14:18:10.122457902 +0200
@@ -1,4 +1,3 @@
-PACKAGE_TARNAME=@PACKAGE_TARNAME@
 prefix=@prefix@
 exec_prefix=@exec_prefix@
 libdir=@libdir@
@@ -6,6 +5,7 @@
 datarootdir=@datarootdir@
 docdir=@docdir@
 doxygendocdir=@htmldir@/doxygen-html
+PACKAGE_TARNAME=@PACKAGE_TARNAME@
 
 Name: @PACKAGE@
 Description: @PKGCONFIG_DESCRIPTION@
